### DISCUSSION TITLE
Larger gravity generator limit

### DISCUSSION THREAD

**Discussion by Sekiberius** [tsunamayo/Starship-EVO#3942] [Suggestions]
My main issue with the current size limit is that when building anything even remotely larger it takes dozens or in the case of larger ships hundreds of gravity generators to fully supply artificial gravity to the entire ship.

Is there anyway the limit could be increased to say 64 or even 100?

**Comment by TIKIRobo** [comment]
personally i think that shipwide gravity would be better because sure we could just keep on increasing the gravity limit or just add a block that gives the whole ship gravity

**Comment by Sekiberius** [comment]
I agree, that'd be the most ideal solution,
The cool thing about the current gravity generators though is that you can create gravity shafts and downwards or upwards boarding hatches. I guess you could have both blocks, one that gives ship wide gravity that has a defined axis that's dependent on placement direction and also have one effects a small, specific area, but you can change the gravity axis.

**Comment by TIKIRobo** [comment]
yea that would work best!

**Comment by Zenti98** [comment]
My suggestion would be to invert the current system. The entire ship has gravity inside of it and there are blocks which suspend the gravity field in a specific area. The problem with one gravity field for an entire ship is, that the inside of the ship may not be clearly defined. There might be many unconventional designes, with modules at the outside of the main hull or whatever...
So it would be nice to define gravity in one big cube and for all exeption areas there are normal bricks which allow to suspend the field inside of this cube - like the opposite of the current system
